### PR TITLE
First merge request - Adding an action hook to the earnings report.

### PR DESCRIPTION
**Pull request by atwellpub**
This is my first fork and merge request on github so forgive me if I've done this wrong. 

I'm wanting to create a hook that will let 3rd party developers perform additional calculations and echo the data under the earnings report graph; so am proposing new action hook called edd_reports_graph_additional_stats.

The appended arrays: dates, sales_data and earnings_data looks valuable for hook considerations. For example I could use $earnings_data and $sales_data to compute an estimated transaction fees paid for the given time and then echo the data into the report.

I am not sure how to docblock action hooks or I would have given it a try. Hope I'm submitting this the right way.

### CODE REVIEW

File: easy-digital-downloads.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Plugin Name: Easy Digital Downloads
+ * Plugin Name: Easy Digital Downloads - change here.
```
Comment (cklosowski) [inline easy-digital-downloads.php]: Might want to remove this change. Looks like testing?

Just remove the changes, and commit then push. The PR will be updated.

**Comment by cklosowski** [comment]
@atwellpub If you can make the necessary changes outlined above, we'll take another look and merge into the master.

**Comment by atwellpub** [comment]
Ok I think the right changes are in and the merge request updated.

**Comment by cklosowski** [comment]
Thanks @atwellpub. Just a minor merge conflict and a formatting to a single line, but looks great.

**Pull request merged**